jest-preset.js

```diff
@@ -0,0 +1,29 @@
+const { defaults } = require('ts-jest/presets')
+
+module.exports = {
+  globals: {
+    'ts-jest': {
+      tsConfig: '<rootDir>/src/tsconfig.spec.json',
+      stringifyContentPathRegex: '\\.html$',
+      astTransformers:[
+        require.resolve('./InlineHtmlStripStylesTransformer')
+      ]
+    },
+  },
+  transform: {
+    '\\.(ts|html)$': 'ts-jest',
```

Comment:
we don't provide transformer for JS files now, is it safe to assume any JS code in 3rd party dependencies is valid ES2015 code?

Reply:
Fixed this one, was taken over from previous commit this PR is built on top of.